### DISCUSSION TITLE
[Feature]: Markdown Editor Update

### DISCUSSION THREAD

**Discussion by rsp2k** [django-wiki/django-wiki#1381] [Ideas]
### I've already checked other feature proposals

Yes

### Is your feature request related to a problem?

No

### Describe your problem.

The included MarkItUp editor seem a bit dated. 

### Describe your feature request.

Searching for license-friendly projects, I found [cherry-markdown: [Demo w/Images](https://tencent.github.io/cherry-markdown/examples/img.html) [GitHub](https://github.com/Tencent/cherry-markdown). Is there any interest, and/or suggestions for other markdown editors?

**Comment by benjaoming** [comment]
See also: https://github.com/django-wiki/django-wiki/discussions/1211

The main concern is to introduce an editor that doesn't play well with extensions and images. But it doesn't have to be a big problem :+1: 

The way that the current editor is implemented is with respect to being able to choose another editor.
